docs/tutorial_human_popgen.md

```diff
@@ -0,0 +1,515 @@
+--preserve5p \
+--bwaalnn 0.01 \
+--run_trim_bam \
+--bamutils_clip_half_udg_left 2 \
+--bamutils_clip_half_udg_right 2 \
+--run_sexdeterrmine \
+--sexdeterrmine_bedfile '../Reference/genome/1240k.sites.bed' \
+--run_nuclear_contamination \
+--contamination_chrom_name 'X' \
+--run_mtnucratio \
+--mtnucratio_header 'MT' \
+--run_genotyping \
+--genotyping_source 'trimmed' \
+--genotyping_tool 'pileupcaller' \
+--pileupcaller_bedfile '../Reference/genome/1240k.sites.bed' \
+--pileupcaller_snpfile '../Datasets/1240k/1240k.snp'
+```
+
+With this, we are ready to submit! If running on a remote cluster/server, Make
+sure to run this in `screen` session or similar, so that if you get a `ssh`
+signal drop or want to log off, Nextflow will not crash.
+
+## Results
+
+Assuming the run completed without any crashes (if problems do occur, check all
+against [#usage](usage.md) all parameters are as expected, or check the
+[FAQ](../faq.md)), we can now check our results in `results/`.
+
+### MultiQC Report
+
+In here there are many different directories containing different output files.
+The first directory to check is the `MultiQC/` directory. In here you should
+find a `multiqc_report.html` file. You will need to view this in a web browser,
+so I recommend either mounting your server to your file browser, or downloading
+it to your own local machine (PC/Laptop etc.).
+
+Once you've opened this you can go through each section and evaluate all the
+results. You will likely want to check these for artefacts (e.g. weird damage
+patterns on the human DNA, or weirdly skewed coverage distributions).
+
+For example, I normally look for things like:
+
+General Stats Table:
+
+- Do I see the expected number of raw sequencing reads (summed across each set
+  of FASTQ files per library) that was requested for sequencing?
+- Does the percentage of trimmed reads look normal for aDNA, and do lengths
+  after trimming look short as expected of aDNA?
+- Does ClusterFactor or 'Dups' look high (e.g. >2 or >10%
+  respectively) suggesting over-amplified or
+  badly preserved samples?
+- Does the human DNA show increased frequency of C>Ts on the 5' end of molecules
+  (you may need to un-hide the 2nd base columns with 'configure columns'
+  button)?
+- Is the number of SNPs used for nuclear contamination really low for any
+  individuals (e.g. < 100)? then the estimates might not be very accurate.
+
+FastQC (pre-AdapterRemoval):
+
+- Do I see any very early drop off of sequence quality scores suggesting
+  problematic sequencing run?
+- Do I see outlier GC content distributions?
+- Do I see high sequence duplication levels?
+
+AdapterRemoval:
+
+- Do I see high numbers of singletons or discarded read pairs?
+
+FastQC (post-AdapterRemoval):
+
+- Do I see improved sequence quality scores along the length of reads?
+- Do I see reduced adapter content levels?
+
+Samtools Flagstat (pre/post Filter):
+
+- Do I see outliers, e.g. with unusually high levels of human DNA, (indicative
```

Comment:
Describe what would be an outlier in your case!

Reply:
Done